snapcraft/linters/library_linter.py

```diff
@@ -87,9 +89,21 @@ def _check_dependencies_satisfied(
         :param root_path: The absolute path to the payload directory.
         :param issues: The list of linter issues.
         """
+        try:
+            linker = elf_utils.get_dynamic_linker(root_path=Path("/"), snap_path=Path())
+            linker_name = Path(linker).name
+        except elf_errors.DynamicLinkerNotFound:
+            linker_name = None
+
+        emit.debug(f"dynamic linker name is: {linker_name}")
```

Comment:
pedantic:
`emit.debug(f"dynamic linker name is: {linker_name!r}")` to show empty quotes when linker is None

or 

```
        try:
            linker = elf_utils.get_dynamic_linker(root_path=Path("/"), snap_path=Path())
            linker_name = Path(linker).name
            emit.debug(f"dynamic linker name is: {linker_name}")
        except elf_errors.DynamicLinkerNotFound:
            linker_name = None
            emit.debug("could not find dynamic linker")
```

Reply:
That's a good one, added `!r` (should be enough for a debug message).